modules/core/arrow-test/src/main/kotlin/arrow/test/laws/MonoidalLaws.kt

```diff
@@ -12,35 +15,38 @@ object MonoidalLaws {
 
   fun <F> laws(
     MDAL: Monoidal<F>,
-    AP: Applicative<F>,
-    EQ: Eq<Kind<F, Tuple2<Int, Int>>>,
+    GENK: GenK<F>,
+    EQ: EqK<F>,
     BIJECTION: (Kind<F, Tuple2<Tuple2<Int, Int>, Int>>) -> (Kind<F, Tuple2<Int, Tuple2<Int, Int>>>),
     ASSOCIATIVE_SEMIGROUPAL_EQ: Eq<Kind<F, Tuple2<Int, Tuple2<Int, Int>>>>
```

Comment:
I don't think this eq is special, so it could be constructed using `EQ`. Can you test removing it and building it with `EqK`?